app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -63,6 +64,7 @@ final void setParticipantService(ParticipantService participantService) {
     public Result getSelfParticipant() throws Exception {
         UserSession session = getAuthenticatedSession();
         Study study = studyService.getStudy(session.getStudyIdentifier());
+        verifySupportedVersionOrThrowException(study);
```

Comment:
Is this necessary? The user will be unable to sign-in if the shut-off switch is turned on. Even if they had existing sessions when the shut-off switch is flipped on, they'll be shut off as soon as their session expires.

If it _is_ necessary, we should add unit tests to make sure the shut-off switch is being considered in these APIs (here and in 2 other places).